src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import { getAllArticles } from "../api";
+import ArticleCard from "./ArticleCard.jsx";
+
+function ArticlesList() {
+  const [allArticles, setAllArticles] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    getAllArticles()
+      .then((response) => {
+        setLoading(false);
+        setAllArticles(response.data.articles);
+      })
+      .catch((error) => {
+        setError(true);
+      });
+  }, []);
+
+  if (error) {
+    return <p>Oh no we can't find your article!</p>;
+  }
+
+  if (loading) {
+    return <p>Loading...</p>;
+  }
+
+  return (
+    <div>
+      {allArticles.map((article) => (
+        <ArticleCard key={article.title} article={article} />
```

Comment:
It's unlikely - but you could have two articles with the same title which would make the keys non-unique
However ids by definition are unique, and we have one for every article!